lighthouse-cli/bin.ts

```diff
@@ -215,7 +215,11 @@ function launchChromeAndRun(addresses: Array<string>,
         return launcher.run();
       })
       .then(() => lighthouseRun(addresses, config, flags))
-      .then(() => launcher.kill());
+      .then(() => launcher.kill())
+      .catch((error) => {
+        console.error(error);
```

Comment:
The stacktrace isnt very user friendly. Let's just log the error using the log method:

`log.error('Lighthouse CLI', error.message);`